src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Microsoft.Extensions.AI.Evaluation.Reporting.csproj

```diff
@@ -0,0 +1,91 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>A library for aggregating and reporting evaluation data. This library also includes support for caching LLM responses.</Description>
+    <TargetFrameworks>$(TargetFrameworks);netstandard2.0</TargetFrameworks>
+    <RootNamespace>Microsoft.Extensions.AI.Evaluation.Reporting</RootNamespace>
+    <!-- EA0002: Use System.TimeProvider to make the code easier to test. -->
+    <NoWarn>$(NoWarn);EA0002</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Workstream>AIEval</Workstream>
+    <Stage>preview</Stage>
+    <ForceLatestDotnetVersions>true</ForceLatestDotnetVersions>
+    <EnablePackageValidation>false</EnablePackageValidation>
+    <MinCodeCoverage>88</MinCodeCoverage>
+    <MinMutationScore>0</MinMutationScore>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Microsoft.Extensions.AI\Microsoft.Extensions.AI.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Extensions.AI.Evaluation\Microsoft.Extensions.AI.Evaluation.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="..\TypeScript\html-report\dist\index.html" Visible="false">
+      <LogicalName>Reporting.HTML.index.html</LogicalName>
+    </EmbeddedResource>
+  </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleToTest Include="Microsoft.Extensions.AI.Evaluation.Integration.Tests" />
+    <InternalsVisibleToTest Include="Microsoft.Extensions.AI.Evaluation.Reporting.Tests" />
+  </ItemGroup>
+
+  <!-- Build the report code before this, so it can be embedded.
+  Setting the BeforeTargets=DispatchToInnerBuilds allows it to build only once before
+  the multiple parallel builds are invoked for each target framework. -->
+  <Target Name="BuildTypescriptBundle"
```

Comment:
Consider moving targets out of proj files into Directory.Build.targets. This would reduce clutter in proj files and could help navigating the code.

Reply:
Good call. Tagging @peterwald - potentially something we can standardize in your follow up PR.